Snakefile

```diff
@@ -138,4 +146,22 @@ rule join_results:
         scenario_paths=postprocessed_paths,
         scenarios=processed_scenarios,
     script:
-        "scripts/join_results.py"
\ No newline at end of file
+        "scripts/join_results.py"
+
+rule analyze_cputime:
+    message:
+        "Time measurement output."
+    input:
+        os.path.join(log_dir, "benchmark-preprocess.log"),  # for Snakemake monitoring only
```

Comment:
Solver time is not listed here, should it be included? If so, would make sense to list it as an output of the postprocessing rule as well.

Reply:
Seems right.